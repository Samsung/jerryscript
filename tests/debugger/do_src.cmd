b f
n
next
n
s
src
n
step
src
c
