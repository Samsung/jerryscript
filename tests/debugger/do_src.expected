Connecting to: localhost:5001
Stopped at tests/debugger/do_src.js:19:1
(jerry-debugger) b f
Breakpoint 1 at tests/debugger/do_src.js:16:3 (in f() at line:15, col:1)
(jerry-debugger) n
Stopped at breakpoint:1 tests/debugger/do_src.js:16:3 (in f() at line:15, col:1)
(jerry-debugger) next
out: F1
Stopped at tests/debugger/do_src.js:20:1
(jerry-debugger) s
Stopped at <unknown>:1:1
(jerry-debugger) source 0
   1 > f = function f() {
   2   print('F2') }
(jerry-debugger) n
Stopped at tests/debugger/do_src.js:21:1
(jerry-debugger) step
Stopped at <unknown>:2:1 (in f() at line:1, col:5)
(jerry-debugger) src
(jerry-debugger) c
out: F2
